Files/App.xaml.cs

```diff
@@ -155,13 +155,13 @@ private static async Task StartAppCenter()
             }
         }
 
-        public static async Task LoadOtherStuffAsync()
+        private static async Task LoadOtherStuffAsync()
```

Comment:
```suggestion
        private static async Task InitializeAppAsync()
```
Do we want to use this opportunity to rename the method entirely?

Reply:
Yeah great idea. When I made this method a while back it was meant to be a temporary name.

Reply:
`InitializeAppComponentsAsync()`